tests/index.test.ts

```diff
@@ -1,14 +1,14 @@
-import { test } from 'tap'
+import { test, TestContext } from 'node:test'
 import { join } from 'path'
 import { readFileSync } from 'fs'
 import Fastify from 'fastify'
 import fastifySession from '@fastify/secure-session'
 import querystring from 'querystring'
 import fastifyFlash from '../src'
 
-const key = readFileSync(join(__dirname, '..', 'secret-key'))
+const key = readFileSync(join(__dirname, '..', '..', 'secret-key'))
 
-test('should set error message and and clear up after displaying.', async t => {
+test('should set error message and and clear up after displaying.', async (t: TestContext) => {
```

Comment:
why does the t variable need to be explicitly typed?

Reply:
Same question as mine and the answer https://github.com/fastify/fastify-flash/pull/110#discussion_r1756199811